website/project/views/node.py

```diff
@@ -49,7 +50,8 @@
 def edit_node(auth, node, **kwargs):
     post_data = request.json
     edited_field = post_data.get('name')
-    value = strip_html(post_data.get('value', ''))
+    value = post_data.get('value', '')
```

Comment:
Although `strip_html` is called in `validate_title`, `Node.set_title` never actually uses the output of this function:  see https://github.com/chennan47/osf.io/blob/feature/edit_title_error_fix/website/project/model.py#L1569

As a result, text is stored in the database exactly as entered. (example sent on flowdock)

Additionally, this change means that no validation or sanitization is performed at all if this function goes down the `description` branch. https://github.com/chennan47/osf.io/blob/feature/edit_title_error_fix/website/project/views/node.py#L62

I've confirmed this by editing the node, than looking at the object from a shell. Can you make sure that both title and description sanitize as appropriate?